use-cases/virgo/synthetic-data-gen/file_gen_hdf5.py

```diff
@@ -0,0 +1,250 @@
+# --------------------------------------------------------------------------------------
+# Part of the interTwin Project: https://www.intertwin.eu/
+#
+# Created by: Jarl Sondre Sæther
+#
+# Credit:
+# - Jarl Sondre Sæther <jarl.sondre.saether@cern.ch> - CERN
+# - Unknown
+# --------------------------------------------------------------------------------------
+
+import argparse
+import sys
+from time import time
+
+import h5py
+import numpy as np
+import pandas as pd
+from gwpy.timeseries import TimeSeries
+from tqdm import tqdm
+
+# sys.path.append(str(Path("..").resolve()))
```

Comment:
Can this be removed?

Reply:
I think so, yes, but I was kind of unsure about which one to keep, haha 😅

Since this is just a script anyway, it probably doesn't matter that much 🤗